src/test/java/com/walmartlabs/x12/dex/dx894/DefaultDex894ParseValidateTest.java

```diff
@@ -92,6 +93,11 @@ public void testParsingValidShipmentMultipleItems() throws IOException {
         // G86 segment
         assertEquals("D83037938029", dexTx.getElectronicSignature());
         assertEquals(null, dexTx.getSignatureName());
+        // transaction data
+        assertNotNull(dexTx.getTransactionData());
+        assertTrue(dexTx.getTransactionData().startsWith("ST*894*148303793"));
+        assertTrue(dexTx.getTransactionData().contains("G83*15*8*EA*001410008561****1.83\r\n"));
+        assertTrue(dexTx.getTransactionData().endsWith("G86*D83037938029\r\n"));
```

Comment:
CRC for String `G86*D83037938029` vs `G86*D83037938029\r\n` will be different?

Reply:
Also, do we need transaction data with trailing `\r\n`?

Reply:
Based on an actual transaction, that seems to be the case.

https://github.com/walmartlabs/gozer/pull/21/files